cmd/template/appcatalog/runner.go

```diff
@@ -87,3 +95,23 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 
 	return nil
 }
+
+// readValuesFromFile reads a configmap from a YAML file.
+func readValuesFromFile(fs afero.Fs, path string) (map[string]string, error) {
+	data, err := afero.ReadFile(fs, path)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	rawMap := map[string]interface{}{}
+	err = yaml.Unmarshal(data, rawMap)
```

Comment:
@corest I am having an error checking the yaml format. Any idea? maybe I am just tired :D

Reply:
How can I reproduce this?

Reply:
You can run appcatalog command like
```
./kubectl-gs template appcatalog --name a --description d --url https://a.com --configmap /tmp/values.yaml
```
Values yaml is just a simple yaml
```
key: value
```